docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: "3.4"
+
+services:
+  production:
+    build: 
+      context: .
+      target: production
+      dockerfile: server/Dockerfile
+    ports: ["8000:80"]
+    environment: 
+      - LOG_LEVEL=DEBUG
+      - GUNICORN_RELOAD=True
+
+  development-server:
+    build: 
+      context: .
+      target: development
+      dockerfile: server/Dockerfile
+    volumes: ["./server:/server"]
+    ports: ["8000:80"]
```

Comment:
This runs the python server for use with the node development server. Note the `target: development` specifying the build target. Also note, if you attempt to visit `localhost:8000` when running `development`, it will fail as `development-server` doesn't contain any of the client code.

Reply:
makes sense that the client code isn't in there during development, however: going to the root of `localhost:8000` in development gave me a 404... i more expected just a blank page with js console errors. dug into this a little bit and i think i figured it out, can you confirm it's because send_static_file is failing in the method?

Reply:
> i more expected just a blank page with js console errors

Why did you expect that?

> can you confirm it's because send_static_file is failing in the method?

Yes, exactly. There is no `index.html` in `/client/build`.

Reply:
i incorrectly expected that because i wasn't thinking that the html file wasn't there, i was thinking that just the JS wasn't there. which was silly.

I am a bit surprised still that it's a 404 that's returned though and not a 500 error or something, because the route does get called... i can see why it's a 404 but i could see it being something else too

Reply:
Hm, 404 seems reasonable, Flask knows there's supposed to be a resource sent, and it's not found -- 404.